core/src/main/java/org/elasticsearch/action/get/MultiGetRequest.java

```diff
@@ -317,32 +314,19 @@ public MultiGetRequest refresh(boolean refresh) {
         return this;
     }
 
-
-    public MultiGetRequest add(@Nullable String defaultIndex, @Nullable String defaultType, @Nullable String[] defaultFields, @Nullable FetchSourceContext defaultFetchSource, byte[] data, int from, int length) throws Exception {
```

Comment:
I removed these because they didn't look used and I was breaking backwards compatibility with them anyway....